h2o-algos/src/main/java/hex/schemas/GLMModelV3.java

```diff
@@ -53,6 +53,12 @@
 
     @API(help="Starting lambda value used when lambda search is enabled.")
     double lambda_max;
+    
+    @API(help="ymu value")
```

Comment:
Would it be possible to add more detailed descriptions to new fields? Sparkling Water in one of the future versions will expose these fields on `H2OMOJOModel` and descriptions in `@API` notation will serve as source for auto-generated documentation.

Reply:
Sure, done.